--- python/lib/python3.7/site-packages/sphinx/themes/basic/static/basic.css_t	2018-10-27 10:35:32.000000000 +0200
+++ python/lib/python3.7/site-packages/sphinx/themes/basic/static/basic.css_t.modified	2018-10-27 10:35:11.000000000 +0200
@@ -54,6 +54,9 @@
     font-size: 90%;
     word-wrap: break-word;
     overflow-wrap : break-word;
+    overflow-y: scroll;
+    top: 0;
+    bottom: 0;
 }
 
 div.sphinxsidebar ul {
